utils/build/docker/dotnet/parametric/query-versions.fsx

```diff
@@ -0,0 +1,75 @@
+#r "nuget: Newtonsoft.Json, 13.0.1"
```

Comment:
This script is exactly the same as `utils/build/docker/dotnet/query-versions.fsx`, why duplicate it ?

Reply:
Makes sense, just removed it from the extra folder and same for the installer one.